.github/workflows/e2e-tests.yml

```diff
@@ -19,17 +19,24 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Set up Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: '20'
+      # Build the Docker image using the Dockerfile in the ./e2e/ folder
+      - name: Build Docker image
+        run: docker build -t playwright-e2e -f ./e2e/Dockerfile .
+
+      # Run the Playwright tests inside the Docker container (without --rm flag)
+      - name: Run E2E tests in Docker container
+        run: |
+          docker run --name playwright-e2e-container -w /app playwright-e2e make e2e-test APP_NAME=${{ inputs.app_name }} BASE_URL=${{ inputs.service_endpoint }}
```

Comment:
are you planning on updating `make e2e-test` to use this command? then we can keep this as `make e2e-test`